frontend/app/views/fragments/nav.scala.html

```diff
@@ -11,30 +12,46 @@
                     <span class="u-h">Home</span>
                 </a>
             </li>
-            <li class="nav__item">
-                <a href="/events" class="nav__link" id="qa-nav-events">Events</a>
-            </li>
-            <li class="nav__item">
-                <a href="/masterclasses" class="nav__link" id="qa-nav-masterclasses">Masterclasses</a>
-            </li>
-            <li class="nav__item">
-                <a href="/about" class="nav__link" id="qa-nav-about">About membership</a>
-            </li>
-            <li class="nav__item">
-                <a href="/patrons" class="nav__link" id="qa-nav-patrons">Patrons</a>
-            </li>
-            <li class="nav__item">
-                <a href="/join" class="nav__link" id="qa-nav-pricing">Pricing</a>
-            </li>
-            <li class="nav__item">
-                <a href="/help" class="nav__link" id="qa-nav-help">Help</a>
-            </li>
-            <li class="nav__item">
-                <a href="/feedback" class="nav__link" id="qa-nav-feedback">Feedback</a>
-            </li>
+            @for(nav <- Nav.navigation) {
+                @if(isFooter || (!isFooter && !nav.footerOnly)) {
+                    <li class="nav__item">
+                        <a href="@nav.href" class="nav__link" id="qa-nav@if(isFooter){-footer}-@nav.id">@nav.title</a>
+                    </li>
+                }
+            }
             <li class="nav__item nav__item--right">
-                <a href="@Config.guardianMembershipUrl" class="nav__link nav__link--last js-members-area is-hidden">Members' Area</a>
+                <a href="@Config.guardianMembershipUrl" class="nav__link nav__link--last@if(!isFooter){ js-members-area} is-hidden">Members' Area</a>
             </li>
         </ul>
     </div>
 </nav>
+
+
+
+@if(!isFooter) {
```

Comment:
Gotcha: Not a huge fan of state/mode flags in fragments. If there's enough difference between the two it might be preferable to split them out into two fragments, I think it's easier to reason about even at the expense of some duplicated markup.